### PR TITLE
Evaluation plots for commit edit types (number two)

### PR DESCRIPTION
**Pull request by clhunsen**
### Prerequisites

- [X] I adhere to the coding conventions (described [here](https://github.com/se-passau/coronet/blob/master/CONTRIBUTING.md)) in my code.
- [X] I have updated the copyright headers of the files I have modified.
- [X] I have written appropriate commit messages, i.e., I have recorded the goal, the need, the needed changes, and the location of my code modifications for each commit. This includes also, e.g., referencing to relevant issues.
- [X] I have put signed-off tags in *all* commits.
- [X] I have updated the changelog file [NEWS.md](https://github.com/se-passau/coronet/blob/master/NEWS.md) appropriately.
- [X] The pull request is opened against the branch `dev`.

### Description

This PR is the same as PR #171, with just the last review comments applied. See there for more details.

Note: I left only PR #171 in the changelog because all credits go to @klaraschlueter! :+1: 

#### Added
- Add a new file `util-plot-evaluation.R` containing functions to plot commit edit types per author and project. (PR #171, d4af515f859ce16ffaa0963d6d3d4086bcbb7377, aa542a215f59bc3ed869cfefbc5a25fa050b1fc9. 0a0a5903e7c609dfe805a3471749eb2241efafe2)

--------------------------

I am not sure whether this PR needs some further reviewing, but I will leave that to someone else to decide, i.e., someone else should merge. :wink:

### CODE REVIEW

**Comment by clhunsen** [comment]
Ah, I missed to update the file `util-init.R`, so I needed to force-push the last commit. Sorry about that.

File: util-plot-evaluation.R [hunk 1]
```diff
@@ -58,9 +58,9 @@ plot.commit.editor.types.by.author = function(data, percentage.per.author = FALS
         value.columns = plot.data[2:4]
 
         ## scale data values per author (represented by one line) to 100%
-        scaled.value.columns = apply(value.columns, 1, function(x) {x/sum(x)})
+        scaled.value.columns = t(apply(value.columns, 1, function(x) {x / sum(x)}))
```
Comment (bockthom) [inline util-plot-evaluation.R]: I am not sure whether we really should move the call to `t` here. Does this still match with the comment above?

Comment (clhunsen) [reply util-plot-evaluation.R]: I applied this change as I added [a review comment on this](https://github.com/se-passau/coronet/pull/171/files#r341076054) to PR #171: The command `t` is moved to streamline the structure of the different variables that are used in that piece of code. I.e., `value.columns` and `scaled.value.columns` have the same structure when this change is applied. :wink:

**Comment by clhunsen** [comment]
Thanks for spotting both of these! :+1: 

> (1) You did not rename the file name in the README file. Please fix that. Maybe just grep for the old file name in the complete directory to make sure that it does not appear anymore else.

I was sure that I missed something... :laughing: 

> (2) I am not sure whether or why we should move the `t` function call. Please see my comment at the corresponding code line.

Please see my comment [above](https://github.com/se-passau/coronet/pull/175#discussion_r373470675).

---------------------------

I rebased the last commit and force-pushed. Feel free to merge if everything is okay, do not hesitate to comment if there are still open issues.

**Pull request merged**